prisma/schema.prisma

```diff
@@ -64,13 +65,17 @@ model CourseContent {
 }
 
 model Certificate {
+  id       String @id @default(cuid())
```

Comment:
?

Reply:
please check now apologies it was my local formatting issue.